tools/downloader/tests/representative-models-quantization.lst

```diff
@@ -1,3 +1,4 @@
 # A list of models for testing quantization features.
 
 mobilenet-v1-0.25-128
+hbonet-0.5
```

Comment:
Please mark these to make it clear why they are here, e.g.:

```
mobilenet-v1-0.25-128 # default config
hbonet-0.5 # custom config
```

Reply:
Done